whois-nrtm4-client/src/test/java/net/ripe/db/nrtm4/client/NrtmServerDummy.java

```diff
@@ -0,0 +1,138 @@
+package net.ripe.db.nrtm4.client;
+
+import com.google.common.io.Resources;
+import jakarta.annotation.PostConstruct;
+import jakarta.annotation.PreDestroy;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import net.ripe.db.nrtm4.client.client.NrtmRestClient;
+import net.ripe.db.whois.common.Stub;
+import net.ripe.db.whois.common.aspects.RetryFor;
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.apache.commons.compress.utils.Lists;
+import org.eclipse.jetty.server.NetworkConnector;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+
+@Profile({WhoisProfile.TEST})
+@Component
+public class NrtmServerDummy implements Stub {
+
```

Comment:
You should not be needing this  if whois server is started and nrtm.emabled is true

Reply:
yes, we are not using an external service...it is the same project. You are right.
Let me refactor this

Reply:
Ok after the discussion on chat, let's keep the stub and mocking nrtmv4 server side. We can change this later on but its fine to use stubs for now